### ISSUE TITLE
Exclusions cause false-negatives

### ISSUE THREAD

**Issue by calvis** [CircleCI-Public/shellcheck-orb#37]
## Orb version:

2.2.2

## What happened:

Here's a minimal reproduction: https://github.com/calvis/shellcheck-orb-bug

From the build output of the `shellcheck-orb-exclusion` job:
```
scripts/bad-script
Invalid number: 1071 
```
The exclusion is passed to shellcheck as `1071 ` (note the space). This syntax error causes shellcheck to abort without checking the scripts, yet the orb reports `No ShellCheck Errors Found` and succeeds.

## Expected behavior:

All three of the jobs in this repository should fail.

## Additional Information:

There's a space where the exclude augment is built: https://github.com/CircleCI-Public/shellcheck-orb/blob/1e15b638c8142accca6c83a29e2768d11c837298/src/scripts/check.sh#L3

The tests for this orb show the same error: https://app.circleci.com/pipelines/github/CircleCI-Public/shellcheck-orb/291/workflows/9ba866dc-66cb-4eb7-bcf9-c6378cb72b32/jobs/1003

**Comment by timoludwig** [comment]
I just tested this out and the trailing space is indeed the culprit.
See:
- [job 13121](https://app.circleci.com/pipelines/github/Integreat/integreat-cms/1836/workflows/5d24a8f1-114c-403b-8693-6f43e097acc9/jobs/13121) (succeeded despite warnings [with trailing space](https://github.com/Integreat/integreat-cms/blob/c3ddcbfb4857b2c4976a8738b699b36c01d4af13/.circleci/config.yml#L70)) vs
- [job 13131](https://app.circleci.com/pipelines/github/Integreat/integreat-cms/1837/workflows/5a1adff2-9ad1-4fc6-be7b-93f093e88e02/jobs/13131) (correctly failing [without trailing space](https://github.com/Integreat/integreat-cms/blob/ffa2513a23b09236a93dcd8ca074c4fb9361dab8/.circleci/config.yml#L70))...
